src/View/DeleteView.js

```diff
@@ -5,6 +5,98 @@ class DeleteView extends View {
         super(name);
         this._type = 'DeleteView';
         this._enabled = true;
+        this._submitCreationSuccess = null;
+        this._submitCreationError = null;
+    }
+
+    /**
+     * Add a function to be executed after the delete succeeds.
+     *
+     * This is the ideal place to use the response to delete the entry, or
+     * redirect to another view.
+     *
+     * If the function returns false, the default execution workflow is stopped.
+     * This means that the function must provide a custom workflow.
+     *
+     * If the function throws an exception, the onSubmitError callback will
+     * execute.
+     *
+     * The syntax depends on the framework calling the function.
+     *
+     * With ng-admin, the function can be an angular injectable, listing
+     * required dependencies in an array. Among other, the function can receive
+     * the following services:
+     *  - $event: the form submission event
+     *  - entry: the current Entry instance
+     *  - entity: the current entity
+     *  - form: the form object (for form validation and errors)
+     *  - progression: the controller for the loading indicator
+     *  - notification: the controller for top notifications
+     *
+     * The function can be asynchronous, in which case it should return
+     * a Promise.
+     *
+     * @example
+     *
+     *     post.deletionView().onSubmitSuccess(['progression', 'notification', '$state', 'entry', 'entity', function(progression, notification, $state, entry, entity) {
+     *         // stop the progress bar
+     *         progression.done();
+     *         // add a notification
+     *         notification.log(`Element #${entry._identifierValue} successfully edited.`, { addnCls: 'humane-flatty-success' });
```

Comment:
Can you amend this message ? It's not the edit view anymore. (same 3 lines below)